aimmo-game/simulation/pickups.py

```diff
@@ -81,16 +80,16 @@ def serialise(self):
         }
 
 
-class DamagePickup(_PickupEffect):
+class DamageBoostPickup(_PickupEffect):
```

Comment:
that's fine but the refactoring could be another pull request as not really linked with test... :)

Reply:
Do we not want to refactor code as we see it?

Reply:
Yes, in another pull request :) Else it's just confusing as the current PR is actually doing 2 or 3 things instead of what is described, and then makes it harder to deal with later. See some interesting things here: https://ro-che.info/articles/2016-09-18-good-pull-requests and there: https://blog.codacy.com/why-youre-not-merging-your-pull-requests-and-what-to-do-about-it-12701c70ae5c

Reply:
I'd rather you remove this from this PR and prepare a small pull request for this refactoring

Reply:
:+1: okk